pyproject.toml

```diff
@@ -10,8 +10,8 @@ packages = [{include = "ray_on_golem"}]
 python = "^3.8"
 ray = {version="^2.7.1", extras=["default"]}
 #golem-core = { path="../golem-core-python", develop=true }
-#golem-core = { git="https://github.com/golemfactory/golem-core-python.git", branch="main" }
-golem-core = "^0.3.2"
+golem-core = { git="https://github.com/golemfactory/golem-core-python.git", branch="main" }
```

Comment:
are we sure we want to merge it into `develop` this way?

Reply:
I think we are